api_v3/services/UserService.php

```diff
@@ -595,6 +595,17 @@ function indexAction($id, $shouldUpdate = true)
 			
 		return $kuser->getPuserId();
 	}
-
+	
+	/**
+	 * Loges a user to the destination account as long the ks user id exists in the desc acount and the loginData id match for both accounts
+	 *
+	 * @action loginByKs
+	 * @param int $destPartnerId
+	 * @throws APIErrors::PARTNER_CHANGE_ACCOUNT_DISABLED
+	 */
+	public function loginByKsAction($destPartnerId)
```

Comment:
You didn't specify the returned type but I understand it's string, I think we would like to avoid new actions that return primitive types if possible.